src/quick-tour/differentiate-personalize.md

```diff
@@ -3,20 +3,20 @@ conditions: Default.B2B Only
+Create Custom Experiences
+:  Magento offers a rich set of tools to create [personalized experiences]({{ site.baseurl }}{% link quick-tour/engage.md %}) across multiple digital touchpoints, based on customer activity and profile.
 
-[Custom Catalog and Price Lists]({{ site.baseurl }}{% link catalog/catalog-shared-pricing-structure.md %})
-:  Offer a curated product selection with custom pricing for specific companies, while continuing to offer the standard catalog with regular pricing for general customers.
+Custom Catalog and Price Lists
+:  Offer a curated product selection with custom [pricing for specific companies]({{ site.baseurl }}{% link catalog/catalog-shared-pricing-structure.md %}), while continuing to offer the standard catalog with regular pricing for general customers.
 
-[Targeted Segmentation]({{ site.baseurl }}{% link marketing/customer-segments.md %})
-:  Optimize your marketing initiatives with dynamic content, promotions, and banner based on properties such as customer address, order history, and shopping cart contents.
+Targeted Segmentation
+:  Optimize your marketing initiatives with dynamic content, promotions, and banner [based on properties]({{ site.baseurl }}{% link marketing/customer-segments.md %}) such as customer address, order history, and shopping cart contents.
 
-[Smart Shopping Rules]({{ site.baseurl }}{% link marketing/promotions.md %})
-:  Customize the shopping experience with price rules and promotions that trigger at the product or shopping cart level.
+Smart Shopping Rules
+:  Customize the shopping experience with price rules and [promotions]({{ site.baseurl }}{% link marketing/promotions.md %}) that trigger at the product or shopping cart level.
 
-[Customer Groups]({{ site.baseurl }}{% link customers/customer-groups.md %})
-:  Offer different products and pricing according to customer group or [shared catalog]({{ site.baseurl }}{% link catalog/catalog-shared.md %}). Determine which discounts are available, and the tax class that applies to the order.
+Customer Groups
+:  Offer different products and pricing according to [customer group]({{ site.baseurl }}{% link customers/customer-groups.md %}) or [shared catalog]({{ site.baseurl }}{% link catalog/catalog-shared.md %}). Determine which discounts are available, and the tax class that applies to the order.
 
-[Maintain Multiple Tailored Websites]({{ site.baseurl }}{% link stores/websites-stores-views.md %})
-:  Maintain multiple tailored sites based on brand, geography, channel partner, or account. Introduce new market and languages, and track analytics from a single Admin.
+Maintain Multiple Tailored Websites
+:  Maintain [multiple tailored sites]({{ site.baseurl }}{% link stores/websites-stores-views.md %}) based on brand, geography, channel partner, or account. Introduce new market and languages, and track analytics from a single Admin.
```

Comment:
this looks like a series of 3.

"...or account, introduce new market and languages, and track analysis...."